applications/tasks/src/modules/kt_numbers.erl

```diff
@@ -259,6 +330,30 @@ account_id(_) -> 'false'.
 carrier_module(Data) ->
     lists:member(Data, knm_carriers:all_modules()).
 
+-spec ported_in(ne_binary()) -> boolean().
+ported_in(Cell) -> is_cell_boolean(Cell).
+
+-spec 'cnam.inbound'(ne_binary()) -> boolean().
+'cnam.inbound'(Cell) -> is_cell_boolean(Cell).
+
+-spec 'prepend.enabled'(ne_binary()) -> boolean().
```

Comment:
I have implemented a couple tasks now and have yet to see a reason for 'function.name' style naming. Why are the alternatives suggested not reasonable?

And read my last sentence again - I fully understand why the tickies are needed; just pointing out that its ironic an opponent of the ticky for atoms would purposefully choose a function name that requires the ticky to work (esp in light of there being alternatives that don't require tickies to be compiled).